commands/internal/hijacker/hijacker.go

```diff
@@ -51,8 +51,8 @@ func (h *Hijacker) Hijack(handle string, spec atc.HijackProcessSpec, pio Process
 
 	dialer := websocket.Dialer{
 		TLSClientConfig: h.tlsConfig,
+		Proxy: http.ProxyFromEnvironment,
```

Comment:
Add proxy to the dialer by using the http.ProxyFromEnvironment function which the function used by the default transport